exercises/practice/clock/.docs/instructions.append.md

```diff
@@ -7,11 +7,11 @@ your Clock type need to work with the == operator. This means that if your
 New function returns a pointer rather than a value, your clocks will
 probably not work with ==.
 
-While the time.Time type in the standard library (https://golang.org/pkg/time/#Time)
+While the time.Time type in the standard library [https://golang.org/pkg/time/#Time](https://golang.org/pkg/time/#Time)
```

Comment:
```suggestion
While the [time.Time](https://golang.org/pkg/time/#Time) type in the standard library
```

The link probably looks better this way.

Reply:
Definitely!
I was trying to limit my changes so I didn't change the label, this is definitely nicer :)